test/scripts/generate-imported-targets-from-snyk.test.ts

```diff
@@ -40,94 +40,6 @@ describe('Generate imported targets based on Snyk data', () => {
       fileName,
     } = await generateSnykImportedTargets({ groupId: GROUP_ID }, [
       SupportedIntegrationTypesToListSnykTargets.GITHUB,
-    ]);
```

Comment:
all those tests are not needs anymore?

Reply:
Reducing the # of tests as they test the same thing really, GH & GHE are tested together already above